### ISSUE TITLE
[Product BUG]: Environment Variables getting unmanaged layers

### ISSUE THREAD

**Issue by dchrist27** [microsoft/coe-starter-kit#486]
Just upgraded last week to Core version 2.1.6 and I am seeing that the new version of the Welcome message flow is failing. It was working just fine in version Core version 2.0.

I have the environment variable defined
![image](https://user-images.githubusercontent.com/52799471/119510796-6dec8480-bd37-11eb-90b1-106aff7350db.png)

but the GUID returned in the script doesn't match.  I am not sure why the script isn't pulling the correct GUID from the environment variable.

![image](https://user-images.githubusercontent.com/52799471/119511129-bc018800-bd37-11eb-8eb1-08b4ff1bec6c.png)

Dave

**Comment by JeneferM-MSFT** [comment]
The group ID shown in the two screenshots is different. Do you know which is correct? 
Can you check to see if you have unmanaged layers on the flow or on the env variable? If so you will need to delete them.

**Comment by dchrist27** [comment]
The correct group ID starts with 00d.   FYI...I am seeing the same issue with another environment variable.  What I see is that the environment variable now uses paramater('env_var_name') the old way used variables('groupID').

I wasn't aware that we had to remove the solution layer from all environment variables prior to upgrading CoE?   I only removed the solution layer from all customized flows.

Now that I am at 2.1.6 in Core Components will I have to wait for the June release or is there a way to update just the environment variables when running the upgrade? 

I do still have the old flow saved and turned off under My Flows.  Can I run the old flow until the June release?

**Comment by JeneferM-MSFT** [comment]
I dont think you need to remove env vars normally. I just ran into this with a customer yesterday and think there may be a bug impacting us. I still need to investigate
I dont think you need to upgrade to fix this. Just remove the unmanaged layer from the env var, that fixed it for my other customer.

**Comment by dchrist27** [comment]
UPDATE--I removed the solution layer from environment variables Admin eMail and from Power Platform Make Office 365 Group.
The latter is used in the Welcome email flow so I cannot immediately test that but I did test against this flow:  Microsoft Teams Admin | Weekly Clean Up of Microsoft Teams environments and the Admin eMail is populating the default entry and not my email address.

![image](https://user-images.githubusercontent.com/52799471/119560741-ea985680-bd69-11eb-8c46-5b53e28a9dab.png)

From flow run:
![image](https://user-images.githubusercontent.com/52799471/119560839-0e5b9c80-bd6a-11eb-92c8-137ff8d60bfe.png)

I will add that I performed a 'save as' and the flow is running from 'my flows' and not from within the governance component.
